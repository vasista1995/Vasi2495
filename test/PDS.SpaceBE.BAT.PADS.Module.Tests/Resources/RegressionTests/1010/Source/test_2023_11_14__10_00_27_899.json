{
  "IdSource": "SPACEACT2:BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeight:67999:240184473:912",
  "Data_flat_Metadata": {
    "Device": "U5435A",
    "ParameterClass": "NON_SPC",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "GoldBallHeight",
    "ParameterUnit": "um",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "67999",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
    "CreatedTimestamp": "2023-11-14T13:00:48Z",
    "UpdatedTimestamp": "2023-11-14T13:49:24Z",
    "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
    "ExportedTimestampUtc": "2023-11-14T13:00:48Z",
    "ExportedTimestamp": "2023-11-14T13:00:48Z",
    "SampleTimestampUtc": "2023-11-14T06:00:48Z",
    "SampleTimestamp": "2023-11-14T13:00:48Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_Height/AU/30um",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/14/2023 13:00:48",
    "ILdsID": 0,
    "SpcComExtern": "CA-Attributes started. Ticket id: 55b5b55f-ef31-48f5-8dcc-d3bdf7eecf89\nCA triggered by violation 1 (Raw value above specification) in channel (67999/912)\n====\nTicketId=55b5b55f-ef31-48f5-8dcc-d3bdf7eecf89 state=PASS executed OCAP=General, HOLDLOT, STOPEQUIPMENTPROCESSING, QUALITYALARMTRIGGER\nCA triggered by violation -1 (Unknown violation description) in channel (-1/-1)",
    "AcceptFlag": "Y",
    "UpdatedBy": "pobtech",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2526351,
    "ViolCount": 2,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "912",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 24.0,
        "SpecLow": 9.0,
        "SpecTarget": 16.5,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYNNNNNNNNNNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "24,6",
        "Median": "24,2",
        "Sigma": "1,43003496460751",
        "Min": "22,9",
        "Max": "26,6",
        "Q1": "23,9",
        "Q3": "25,4"
      },
      "ControlLimits": {
        "ctrl_mean_low": 11.0,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 17.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 23.0,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.0,
        "ctrl_range_high": 15.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 21.2825,
        "ExtEwmaMV": 22.0800068920489,
        "ExtMsMV": 4.66690475583121,
        "EwmaS": 1.28999146946896,
        "EwmaR": 3.37494767748801,
        "ExtEwmaMVLCL": 12.8656531255506,
        "ExtEwmaMVCenter": 15.268504,
        "ExtEwmaMVUCL": 17.6713548744494,
        "ExtMaMVLCL": 13.5694318513612,
        "ExtMaMVCenter": 15.268504,
        "ExtMaMVUCL": 16.9675761486388,
        "EwmaSLCL": 0.572474202852457,
        "EwmaSCenter": 1.51113357572865,
        "EwmaSUCL": 2.44979294860484,
        "EwmaRLCL": 1.38119890445484,
        "EwmaRCenter": 3.62764,
        "EwmaRUCL": 5.87408109554516,
        "ExtMSMVLCL": 0.00259778821504498,
        "ExtMSMVCenter": 1.2250389281796,
        "ExtMSMVUCL": 4.92102881621818,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 5.64136422726342,
        "RawUal": 24.8956437727366,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240184473,
        "Seqnr": 1,
        "Value": 22.9,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA"
      },
      {
        "SampleId": 240184473,
        "Seqnr": 2,
        "Value": 24.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA"
      },
      {
        "SampleId": 240184473,
        "Seqnr": 5,
        "Value": 25.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA"
      },
      {
        "SampleId": 240184473,
        "Seqnr": 4,
        "Value": 26.6,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA"
      },
      {
        "SampleId": 240184473,
        "Seqnr": 3,
        "Value": 23.9,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 63 TO 60\nTECH : DITA"
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:GoldBallHeight:67999",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeight:67999:240184473:912"
  }
}