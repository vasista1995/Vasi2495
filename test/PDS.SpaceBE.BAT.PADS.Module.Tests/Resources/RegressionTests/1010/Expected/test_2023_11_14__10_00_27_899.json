{
  "Data_flat_Metadata": {
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T06:00:48Z",
    "BeginTimestamp": "2023-11-14T13:00:48Z",
    "EndTimestampUtc": "2023-11-14T06:00:48Z",
    "EndTimestamp": "2023-11-14T13:00:48Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "U5435A",
    "ParameterClass": "NON_SPC",
    "Segment": "WB"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 1,
          "SampleId": 240184473,
          "Value": 22.9,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:00:48Z",
          "SampleTimestampUtc": "2023-11-14T06:00:48Z",
          "CreatedTimestamp": "2023-11-14T13:00:48Z",
          "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
          "UpdatedTimestamp": "2023-11-14T13:49:24Z",
          "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "SampleMean": 24.6,
          "SampleMin": 22.9,
          "SampleMax": 26.6,
          "SampleMedian": 24.2,
          "SampleStdev": 1.43003496460751,
          "SampleQ1": 23.9,
          "SampleQ3": 25.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 2,
          "SampleId": 240184473,
          "Value": 24.2,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:00:48Z",
          "SampleTimestampUtc": "2023-11-14T06:00:48Z",
          "CreatedTimestamp": "2023-11-14T13:00:48Z",
          "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
          "UpdatedTimestamp": "2023-11-14T13:49:24Z",
          "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "SampleMean": 24.6,
          "SampleMin": 22.9,
          "SampleMax": 26.6,
          "SampleMedian": 24.2,
          "SampleStdev": 1.43003496460751,
          "SampleQ1": 23.9,
          "SampleQ3": 25.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 240184473,
          "Value": 25.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:00:48Z",
          "SampleTimestampUtc": "2023-11-14T06:00:48Z",
          "CreatedTimestamp": "2023-11-14T13:00:48Z",
          "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
          "UpdatedTimestamp": "2023-11-14T13:49:24Z",
          "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "SampleMean": 24.6,
          "SampleMin": 22.9,
          "SampleMax": 26.6,
          "SampleMedian": 24.2,
          "SampleStdev": 1.43003496460751,
          "SampleQ1": 23.9,
          "SampleQ3": 25.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 240184473,
          "Value": 26.6,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:00:48Z",
          "SampleTimestampUtc": "2023-11-14T06:00:48Z",
          "CreatedTimestamp": "2023-11-14T13:00:48Z",
          "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
          "UpdatedTimestamp": "2023-11-14T13:49:24Z",
          "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "SampleMean": 24.6,
          "SampleMin": 22.9,
          "SampleMax": 26.6,
          "SampleMedian": 24.2,
          "SampleStdev": 1.43003496460751,
          "SampleQ1": 23.9,
          "SampleQ3": 25.4,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 240184473,
          "Value": 23.9,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:00:48Z",
          "SampleTimestampUtc": "2023-11-14T06:00:48Z",
          "CreatedTimestamp": "2023-11-14T13:00:48Z",
          "CreatedTimestampUtc": "2023-11-14T06:00:48Z",
          "UpdatedTimestamp": "2023-11-14T13:49:24Z",
          "UpdatedTimestampUtc": "2023-11-14T06:49:24Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
          "SampleMean": 24.6,
          "SampleMin": 22.9,
          "SampleMax": 26.6,
          "SampleMedian": 24.2,
          "SampleStdev": 1.43003496460751,
          "SampleQ1": 23.9,
          "SampleQ3": 25.4,
          "SampleSize": 5
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "912",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 24.0,
          "spec_low": 9.0,
          "target": 16.5
        },
        "ControlLimits": {
          "ctrl_mean_low": 11.0,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 17.0,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 23.0,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 0.0,
          "ctrl_range_high": 15.0,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 24.6,
        "median": 24.2,
        "stdev": 1.2790621564255593,
        "range": 3.700000000000003,
        "min": 22.9,
        "max": 26.6,
        "q02": 22.979999999999997,
        "q05": 23.1,
        "q25": 23.9,
        "q75": 25.4,
        "q95": 26.360000000000003,
        "q98": 26.503999999999998,
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
        "ViolationList": "1, 3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 63 TO 60 TECH : DITA",
        "Samples": "240184473"
      },
      "ParameterName": "GoldBallHeight",
      "ParameterUnit": "um",
      "ChannelId": "67999",
      "SourceDataLevel": "C",
      "channel_name": "Ball_Height AU 30um",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T06:00:48Z",
      "created_timestamp": "2023-11-14T13:00:48Z",
      "updated_timestamp_utc": "2023-11-14T06:49:24Z",
      "updated_timestamp": "2023-11-14T13:49:24Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T06:49:24Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeight:67999:240184473:912",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T13:00:48Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}